CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Change Log
 
+## [0.27.0](https://github.com/redhat-developer/vscode-xml/milestone/36?closed=1) (May 27, 2024)
+
+### Enhancements
+
+ * Align XML attributes when formatting. See [#272](https://github.com/redhat-developer/vscode-xml/issues/272).
```

Comment:
Add [the setting feature from lemminx](https://github.com/eclipse/lemminx/pull/1644/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR7)

Reply:
Angelo mentioned that https://github.com/redhat-developer/vscode-xml/issues/945#issuecomment-1836537103 fixed it. I mention it under bug fixes. I could reword it and move it up to enhancements.

Reply:
Okay. Sounds good.